tests/publish/ninjs_formatter_test.py

```diff
@@ -311,7 +311,7 @@ def test_item_with_usable_associations(self):
         formatted = json.loads(doc)
         self.assertIn('associations', formatted)
         self.assertIn('image', formatted['associations'])
-        image = formatted['associations']['image']
+        image = formatted['associations']['image'][0]
```

Comment:
Each value in the associations dict is a list of dicts (see here: https://github.com/superdesk/superdesk-core/blob/master/superdesk/publish/formatters/ninjs_formatter.py#L146)